backend/src/file/file.service.ts

```diff
@@ -124,6 +124,26 @@ export class FileService {
     };
   }
 
+  async removeAndDeleteOne(shareId: string, fileId: string) {
+    const fileMetaData = await this.prisma.file.findUnique({
+      where: { id: fileId },
+    });
+
+    if (!fileMetaData) throw new NotFoundException("File not found");
+
+    fs.unlinkSync(`${SHARE_DIRECTORY}/${shareId}/${fileId}`);
+
+    await this.prisma.file.delete({ where: { id: fileId } });
+
+    return {
```

Comment:
I think the return value of this function isn't used anywhere so we could make the function void